grype/db/curator.go

```diff
@@ -28,11 +30,18 @@ const (
 	FileName = grypeDB.VulnerabilityStoreFileName
 )
 
+var (
+	ErrDataIsStale = errors.New("data is stale")
+
+	DefaultStalenessThreshold = time.Hour * 24 * 5
+)
+
 type Config struct {
 	DBRootDir           string
 	ListingURL          string
 	CACert              string
 	ValidateByHashOnGet bool
+	StalenessThreshold  time.Duration
```

Comment:
It's not immediately obvious what "staleness threshold" means. We should be exact here. We want this value to represent the max time allowed since the database was built.